apps/onnx/onnx_converter_generator.cc

```diff
@@ -0,0 +1,98 @@
+#include "Halide.h"
+#include "onnx_converter.h"
+#include <fstream>
+#include <map>
+
+namespace {
+
+// TODO: This should be an onnx_converter API.
+Halide::Type get_halide_type(const Tensor &tensor) {
```

Comment:
May as well move this into onnx_converter now.

Reply:
I have a follow up PR for updating onnx_converter (to sync with internal facebook).Let's move this in that PR.

Reply:
sgtm